src/App.css

```diff
@@ -1,38 +1,253 @@
+  margin: 0 auto;
+  max-width: 100vw;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+.header-wrapper {
+  width: 90%;
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-between;
+}
+
+h1 {
+  color: #FFFFFF;
+  font-family: 'Alegreya Sans SC', sans-serif;
+  font-size: 4rem;
+  justify-content: center;
+  display: inline-block;
+}
+
+.header h2 {
+  font-style: italic;
+  margin: auto 0;
+  font-size: 1.5rem;
+  display: inline-block;
+}
+
+div.cart-counter {
+  display: flex;
+}
+
+.cart-counter p {
+  margin: auto 0;
+  width: auto;
+  text-align: right;
+  font-size: 1.4rem;
+  display: inline-block;
+}
+
+.cart-counter img {
+  height: 30px;
+  width: auto;
+  margin: auto 30px auto 10px;
+  padding: 5px;
+  display: inline-block;
+}
+
+.wrapper {
+  display: flex;
+  justify-content: space-between;
+  width: calc(90%);
+  margin: 0 auto;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+ul.inventory {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+  padding: 0;
+  margin: 50px auto;
+  width: calc(70%);
+  height: 100%;
+}
+
+li.inventory {
+  list-style: none;
+  padding: 1rem;
+  margin: 1.5rem 5px;
+  width: 160px;
+  height: 200px;
+  text-align: center;
   display: flex;
   flex-direction: column;
+  justify-content: space-around;
   align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+  border: 1px solid #2EC4B6;
+  border-radius: 10px;
+}
+
+.add-to-cart {
+ width: 80%;
+ margin: 0 auto;
+ padding: 5px;
+ background-color: #FFBF69;
+ box-shadow: 1px 1px 10px #FF9F1C;
+ font-family: 'Halant', serif;
+ text-transform: uppercase;
+ letter-spacing: 1px;
+ border-radius: 10px;
+}
+
+p.inventory {
+  font-size: 1.2rem;
+}
+
+.product-name {
+  font-size: 1.3rem;
+  margin: 30px 0 0 0;
 }
 
-.App-link {
-  color: #61dafb;
+.price {
+  margin: 0;
 }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
+p {
+  font-family: 'Halant', serif;
+}
+
+.cart {
+  width: calc(28%);
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+  max-height: 80vh;
+  margin: 15px;
+}
+
+.cart h3 {
+  font-family: 'Iceberg', cursive;
+  width: 50%;
+  margin: 60px auto 0 auto;
+  border-bottom: 1px solid #FF9F1C;
+}
+
+.cart-container {
+  padding-left: 0;
+}
+
+.cart-list {
+  display: flex;
+  margin: 10px 0;
+  justify-content: space-around
+}
+
+.cart-list p {
+  text-align: left;
+  margin: auto 0;
+  display: inline-block;
+  font-size: 1.1rem;
+  width: 250px;
+}
+
+p.cart-price {
+  display: inline-block;
+  width: 100px;
+  text-align: end;
+  margin-right: 10px;
+}
+
+.empty {
+  text-transform: uppercase;
+}
+
+.remove {
+  background-color: #FFFFFF;
+  font-family: 'Halant', serif;
+  text-transform: uppercase;
+  letter-spacing: 1px;
+  font-size: 1rem;
+  border: none;
+  border-radius: 50%;
+  max-height: 26px;
+}
+
```

Comment:
Media queries look great overall! Only small issue I was having is on phone mode, some of the header elements are misbehaving a little, perhaps because of margins. Specifically it looks like the cart image, and the header wrapper have some margins that create a horizontal scroll at that size.